### DISCUSSION TITLE
Signalement dumps en cours d'utilisation

### DISCUSSION THREAD

**Discussion by souhoc** [wow0000/friends42#5] [Ideas]
Salut,

J'ai remarqué qu'il est impossible de report un dump si quelqu'un est log dessus. Ce qui pourrait être utile lorsque le dump freeze (ce qui arrive bcp à MADE)
Pour éviter les abus peut-être restreindre à la personne log seulement ?

**Comment by wow0000** [comment]
Hello,
L'idée n'est pas mauvaise mais les dumps freeze sont assez rare parce que rapidement traité par le staff, il y aurait aussi le risque d'abus que tu as mentionné, malheureusement les PC freeze sont signalé par une minorité des utilisateurs du site (actuellement 141 signalements signalé par 32 personnes) ce qui rends la solution proposé peu-intéressante
Il y a également un problème d'UX j'ai du mal à voir son intégration sur mobile sans surcharger l'interface
Si tu as d'autres idées n'hésites pas ! :)
